security-policy.md

```diff
@@ -1,45 +1,65 @@
+intro: We take security issues very seriously and all the vulnerability reports are treated with the highest priority and confidentiality.
 permalink: /security-policy/
 redirect_from:
     - /security/
     - /security
     - /security-policy
 ---
 
-## Reporting a vulnerability
+Thank you for taking the time to [responsibly disclose](https://en.wikipedia.org/wiki/Responsible_disclosure) any vulnerabilities you find.
 
-Ensure you are using the latest Ballerina version before you run an automated security scan or perform a penetration test against it.
+## Reporting a Vulnerability
 
-Based on the ethics of responsible disclosure, you must **only use the [security@ballerina.io](mailto:security@ballerina.io) mailing list** to report security vulnerabilities and any other concerns regarding the security aspects of the source code or any other resource in this repo.
+First of all, make sure that you are using the latest Ballerina version before you run an automated security scan or perform a penetration test against it. All security vulnerabilities and any other concerns related to security should be reported by sending an emailing to [security@ballerina.io](mailto:security@ballerina.io).  
 
-> **WARNING:** To protect the end-user security, **please do not use any other medium to report security vulnerabilities**. Also, kindly refrain from disclosing the vulnerability details you come across with other individuals, in any forums, sites, or other groups - public or private before it’s mitigation actions and disclosure process are completed.
+> **WARNING:** To ensure end-user security, **please do not use any other medium to report security vulnerabilities**. Also, kindly refrain from disclosing the vulnerability details you come across with others, in any forums, sites, or other groups (public or private) before its mitigation and disclosure processes are completed.
 
-Use the following key to send secure messages to [security@ballerina.io](mailto:security@ballerina.io):
 
-> security@ballerina.io: 0168 DA26 2989 0DB9 4ACD 8367 E683 061E 2F85 C381 [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381)
+If you would like, you can encrypt your report using the  following public key to send secure messages to [security@ballerina.io](mailto:security@ballerina.io):
 
-Also, use the following template when reporting vulnerabilities so that it contains all the required information and helps expedite the analysis and mitigation process.
+> security@ballerina.io: 0168 DA26 2989 0DB9 4ACD 8367 E683 061E 2F85 C381 
 
-- **Vulnerable Ballerina artifact(s) and version(s):** list of the Ballerina artifacts/versions that are considered as vulnerable
-- **Overview:** high-level overview of the issue and self-assessed severity
+This key can also be found at [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381).
```

Comment:
`pgp.surfnet.nl` isn't a well know. Let's please upload the public key also to `https://keys.openpgp.org/` and get the openpgp.org link listed here. 

In addition, shall we discuss with @anupama-pathirage if it's fine for the security team to manage the key in this instance (since security team is managing key for all other security reporting channels operated within WSO2). If agreeable, we'll re-issue a key-pair and share the new key with @anupama-pathirage (and take care of managing the key from that point onward).

Reply:
+1 for that option. Also, the existing key is already expired and we may have to regenerate it again. In that case, it would be better to manage a new key-pair by security team going forward IMO.
@anupama-pathirage Shall we proceed with that?

Reply:
Yes +1

Reply:
@ayomawdb Let's proceed with this. We may have to update both ballerina.io/security and wso2.com/security pages along with this.

Reply:
Ack.. I'll share a the public-key over here.

Reply:
Links to new public key can be found in https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Reporting+Guidelines

Reply:
Thanks @ayomawdb for the prompt action.

Reply:
@praneesha let's update this as 'This key can also be found at [keys.openpgp.org](https://keys.openpgp.org/vks/v1/by-fingerprint/AC483C56C0A060204BBEF3E4182F3F21255FCCE9).

Reply:
Fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/3095/commits/4ea2ceb06d5078e86614848ffef76c7731ebd8cb.